airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/commands.py

```diff
@@ -47,13 +53,15 @@
     type=click.Choice([step_id.value for step_id in CONNECTOR_TEST_STEP_ID]),
     help="Skip a step by name. Can be used multiple times to skip multiple steps.",
 )
+@click.argument("extra_params", nargs=-1, type=click.UNPROCESSED)
```

Comment:
❓ Should we combine this decorator and `build_extra_params_mapping` into a custom decorator

```
@allow_extra_params
```

That will parse the raw `extra_params` given by click into a usable dict?